File: modules/wem-api/src/main/java/com/enonic/wem/api/query/aggregation/Bucket.java
package com.enonic.wem.api.query.aggregation;

public class Bucket
{
    final String key;

    final long docCount;

    public Bucket( final String key, final long docCount )
    {
        this.key = key;
        this.docCount = docCount;
    }

    public String getKey()
    {
        return key;
    }

    public long getDocCount()
    {
        return docCount;
    }
}

File: modules/wem-webapp/src/main/webapp/admin/common/js/aggregation/Bucket.ts
module api.aggregation {

    export class Bucket {

        private key: string;
        private docCount: number;

        public getKey(): string {
            return this.key;
        }

        public getDocCount(): number {
            return this.docCount;
        }

        public static fromJson(json: api.aggregation.BucketJson): Bucket {
            var bucket: Bucket = new Bucket();
            bucket.key = json.key;
            bucket.docCount = json.docCount;
            return bucket;
        }
    }
}